cea/inputlocator.py

```diff
@@ -468,6 +474,33 @@ def get_edge_mass_flow_csv_file(self, network_type):
         return os.path.join(self.get_optimization_network_layout_folder(), 'NominalEdgeMassFlow_' +
                              network_type + '.csv')
 
+    def get_network_layout_edges_shapefile(self, network):
```

Comment:
The  `reference-case-open` now includes the .shp files of network nodes/edges under this directory: `...baseline\inputs\thermal-networks\DC or DH`. In `DC`, a network with single plant is set as default, while `DH` has network with two plants. The rest of the network `.shp` files are stored in `validation\thermal_network_matrix\test_networks_open_case`.